app/controllers/application_controller.rb

```diff
@@ -74,9 +75,10 @@ def metrics_logger
   end
 
   def ensure_visit_integrity
-    unless visit && visit.prisoner && visit.prisoner.prison_name.present?
+    unless visit && visit.prisoner && visit.prison_name.present?
       redirect_to edit_prisoner_details_path,
-        notice: 'You need to complete missing information to start or continue your visit request'
+        notice: 'You need to complete missing information to start' \
```

Comment:
The same goes for the other long strings of English text in this PR, by the way.